icu4c/source/common/cjmlbe.cpp

```diff
@@ -0,0 +1,457 @@
+// © 2022 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+#include <utility>
+
+#include "unicode/utypes.h"
+
+#if !UCONFIG_NO_BREAK_ITERATION
+
+#include "cjmlbe.h"
+#include "brkeng.h"
+#include "dictbe.h"
+#include "unicode/uniset.h"
+#include "unicode/chariter.h"
+#include "unicode/resbund.h"
+#include "unicode/ubrk.h"
+#include "unicode/usetiter.h"
+#include "ubrkimpl.h"
+#include "utracimp.h"
+#include "uvectr32.h"
+#include "uvector.h"
+#include "uassert.h"
+#include "unicode/normlzr.h"
+#include "cmemory.h"
+#include "dictionarydata.h"
+#include "util.h"
+
+U_NAMESPACE_BEGIN
+
+#if !UCONFIG_NO_NORMALIZATION
```

Comment:
why do we need to wrap around #if !UCONFIG_NO_NORMALIZATION for this. Your code does not use normalization so why is the need for wrap with this?

Reply:
CjkBreakEngine is wrapped in !UCONFIG_NO_NORMALIZATION and then initiated CjMLEngine.
If so, shall I wrap CjMLEngine in !UCONFIG_NO_NORMALIZATION?

Reply:
CjkBreakEngine is wrapped in !UCONFIG_NO_NORMALIZATION because it use normalizer. If it is not wrap UCONFIG_NO_NORMALIZATION then in the complation time, if that flag is set (because the user want to strip off normalization, then that part of code will not compile. But for your case, CjMLEngine has no dependency on  normalizer. Your code should be wrap inside #if !UCONFIG_NO_BREAK_ITERATION but not  !UCONFIG_NO_NORMALIZATION

Reply:
Done